cms/templates/flexiblepricing/flexible_pricing_request_form.html

```diff
@@ -4,48 +4,58 @@
 {% block title %}{{ site_name }} | {{ page.title }}{% endblock %}
 
 {% block content %}
+<div id="main" class="main-page-content">
+  <div class="std-page-body container auth-page">
+    <div class="auth-card card-shadow auth-form">
+    
+      <div class="auth-header">
+          <h1 class="text-uppercase" id="{{ page.title|slugify }}">Request Flexible Pricing</h1>
 
-  <div id="main" class="resource-banner">
-    <div class="container">
-      <h1 class="text-uppercase">{{ page.title }}</h1>
-      {% if page.sub_heading %}
-        <h2>{{ page.sub_heading }}</h2>
-      {% endif %}
-    </div>
-  </div>
-
-<div class="container resource-container">
-  <div class="row">
-    <div class="col-md-2">
-    </div>
-    <div class="col-md-8">
-        <h2 class="text-uppercase" id="{{ page.title|slugify }}">{{ page.title }}</h2>
-
-        {% if course is not None %}
-        <p>Requesting flexible pricing for {{ course.product.title }} ({{ course.product.readable_id }}).</p>
-        {% endif %}
+          {% if course is not None %}
+          <p>Requesting flexible pricing for {{ course.product.title }} ({{ course.product.readable_id }}).</p>
+          {% endif %}
 
-        {{ page.intro|richtext }}
-        
-        {% if user.is_authenticated %}{# is logged in #}
-          {% if has_submission %}{# has submission #}
-            {% if prior_request.is_approved %}{# request status #}
-              {{ page.application_approved_text|richtext }}
-            {% elif prior_request.is_denied %}
-              {{ page.application_denied_text|richtext }}
+          {{ page.intro|richtext }}
+          
+          {% if user.is_authenticated %}{# is logged in #}
+            {% if has_submission %}{# has submission #}
+              {% if prior_request.is_approved %}{# request status #}
+                <b>{{ page.application_approved_text|richtext }}</b>
+              {% elif prior_request.is_denied %}
+                <b>{{ page.application_denied_text|richtext }}</b>
+              {% else %}
+                <b>{{ page.application_processing_text|richtext }}</b>
```

Comment:
Do we want to bold these by default? This is a rich text field so thinking someone may put one of these together and then get confused as to why their text is bold when they didn't specify that.